src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_editor.cljs

```diff
@@ -186,86 +184,113 @@
+        (config "name")]
+       [:div {:style {:display (when-not editing? "none")}}
+        (style/create-text-field {:ref "confname" :defaultValue (config "name")})])
+     (create-section-header "Inputs")
+     (create-section
+       [:div {}
+        (for [key (keys (config "inputs"))]
+          [:div {:style {:verticalAlign "middle"}}
+           [:div {:style {:float "left" :marginRight "1em" :padding "0.5em" :marginBottom "0.5em"
+                          :backgroundColor (:background-gray style/colors)
+                          :border (str "1px solid " (:line-gray style/colors)) :borderRadius 2}}
+            (str key ":")]
+           [:div {:style {:float "left" :display (when editing? "none") :padding "0.5em 0 1em 0"}}
+            (get (config "inputs") key)
+            (validation-status (contains? invalidInputs key))]
+           [:div {:style {:float "left" :display (when-not editing? "none")}}
+            (style/create-text-field {:ref (str "in_" key) :defaultValue (get (config "inputs") key)})]
+           (clear-both)
+           [:div {:style {:marginRight "1em" :padding "0.5em" :marginBottom "0.5em"
+                          :backgroundColor (:exception-red style/colors)
+                          :display (if-not (contains? invalidInputs key) "none")
+                          :width "450px"
+                          :border (str "1px solid " (:line-gray style/colors)) :borderRadius 2}}
+            (get invalidInputs key)]])])
+     (create-section-header "Outputs")
+     (create-section
+       [:div {}
+        (for [key (keys (config "outputs"))]
+          [:div {:style {:verticalAlign "middle"}}
+           [:div {:style {:float "left" :marginRight "1em" :padding "0.5em" :marginBottom "0.5em"
+                          :backgroundColor (:background-gray style/colors)
+                          :border (str "1px solid " (:line-gray style/colors)) :borderRadius 2}}
+            (str key ":")]
+           [:div {:style {:float "left" :display (when editing? "none") :padding "0.5em 0 1em 0"}}
+            (get (config "outputs") key)
+            (validation-status (contains? invalidOutputs key))]
+           [:div {:style {:float "left" :display (when-not editing? "none")}}
+            (style/create-text-field {:ref (str "out_" key) :defaultValue (get (config "outputs") key)})]
+           (clear-both)
+           [:div {:style {:marginRight "1em" :padding "0.5em" :marginBottom "0.5em"
+                          ;TODO: this color is kind of offensive to the eyes on the background, find a friendlier color
+                          :backgroundColor (:exception-red style/colors)
+                          :display (if-not (contains? invalidOutputs key) "none")
+                          :width "450px"
+                          :border (str "1px solid " (:line-gray style/colors)) :borderRadius 2}}
+            (get invalidOutputs key)]])])
+     (create-section-header "Prerequisites")
```

Comment:
Remove this section. We're not using prerequisites, so you might as well take it out in this PR.